person.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+require_relative 'nameable'
+
+# for person
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name: 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+end
+
+def add_rentals(_person, date)
+  Rental.new(date, book, self)
+end
```

Comment:
- [ ]  It appears that your add_rentals method in the Person class is currently defined outside of the class scope. Could you please move it inside the Person class for better encapsulation? Additionally, remember to include :rentals in the attr_accessor list, like this: attr_accessor :name, :age, :rentals. Thank you! 🛠️👷‍♂️